src/components/activity-indicator-overlay/ActivityIndicatorOverlay.tsx

```diff
@@ -23,4 +26,7 @@ const useStyles = StyleSheet.createThemeHook(() => ({
     backgroundColor: 'black',
     opacity: 0.4,
   },
+  activityIndicator: {
+    backgroundColor: theme.static.background['background_accent_0'].text,
+  },
```

Comment:
The overlay is always a black tint. Would it be better with a hardcoded white spinner, to ensure it always works well with the dark overlay? I know `background_accent_0` as of now has white text color for all transport companies, but there is no guarantee that this will always be the case.

Reply:
I see. I´ll update the spinner to use a hardcoded white colour instead👍